### PR TITLE
Updating the types topic according to Architecture standard

### PR DESCRIPTION
**Pull request by GiselleLeeOSI**
1. Remove SDS prefixes where deemed extraneous
2. Correct overcapitalization

### CODE REVIEW

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 1]
```diff
@@ -5,53 +5,56 @@ uid: sdsTypes
...
-events. Events are stored in SdsStreams (or streams).
-An SdsType is a primitive type available in OCS used within the API and REST definitions to identify a stream type.
+events. Events are stored in streams.
+A primitive type is available in the OCS and is used within the API and REST definitions to identify a stream type.
 
-Define simple atomic types, such as integers, floats, strings, arrays, and dictionaries, or 
+You can define simple atomic types, such as integers, floats, strings, arrays, and dictionaries, or 
 complex or nested types using the [Properties collection of SdsTypes](#sdstypeproperty). 
 
-An SdsType used to define an SdsStream must have a key. A key is a [Property, or a combination of Properties](#sdstypeproperty)
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:14]: 'A' not 'An type'

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 2]
```diff
@@ -157,35 +160,40 @@ Version                 | 22
 VersionArray            | 222
 
 ## SdsTypeProperty
-The Properties collection defines the fields in an SdsType. Stream properties are a field expected to appear in every stream created from a given type.
+The `Properties` collection defines the fields in a type. 
+Stream properties are a field expected to appear in every stream created from a given type.
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:164]: This doesn't sound right to me... Maybe 'Stream properties are expected to appear in every stream created from a given type.' But even that sounds weird. Let's brain storm on this one.

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 3]
```diff
@@ -157,35 +160,40 @@ Version                 | 22
...
 SdsTypeProperty is called a compound index. The maximum number of properties that can define a compound index is three. 
-In a compound index, each SdsTypeProperty that is included in the index is specified as ``IsKey``.
+In a compound index, each `SdsTypeProperty` that is included in the index is specified as ``IsKey``.
 The ``Order`` field marks the order of comparison within a compound index.
 
+**`Value`**  
 The ``Value`` field is used for properties that represent a value. An example of a property with a 
-value is an enum's named constant. When representing an enum in an SdsType, the SdsType's 
+value is an enumeration's named constant. When representing an enum in an SdsType, the SdsType's 
 SdsTypeProperty collection defines the enum's constant list. The SdsTypeProperty's ``Id`` represents
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:193]: I think this need some improvement. It's overloaded and not clear. Let's talk about it.

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 4]
```diff
@@ -296,12 +305,12 @@ unique identifiers. Note that the following table contains only a partial list o
 The `SdsTypeBuilder` also supports derived types. Note that you need not add the base types to 
 the SDS before using `SdsTypeBuilder`. Base types are maintained within the SdsType.
 
-## Work with SdsTypes outside of .NET framework
-SdsTypes must be built manually when .NET `SdsTypeBuilder` is unavailable. Below, you'll see how types are built and defined in
+## Work with types outside of .NET framework
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:308]: Should this be 'working' instead? What are your thoughts?

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 5]
```diff
@@ -296,12 +305,12 @@ unique identifiers. Note that the following table contains only a partial list o
...
+Types must be built manually when .NET `SdsTypeBuilder` is unavailable. Below, you'll see how types are built and defined in
 [Python](https://github.com/osisoft/sample-ocs-waveform-python) and [JavaScript](https://github.com/osisoft/sample-ocs-waveform-nodejs) samples. 
 For samples in other languages, go to [OCS code samples in GitHub](https://github.com/osisoft/OSI-Samples-OCS/blob/master/docs/SDS_WAVEFORM_README.md).
 
-SdsType, SdsTypeProperty, and SdsTypeCode are defined below:
+Type, SdsTypeProperty, and SdsTypeCode are defined below:
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:313]: I would leave SdsType here

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 6]
```diff
@@ -590,7 +599,7 @@ class Derived(Simple):
         self.__observation = observation
 ```
 
-Extend the above SdsType as follows:
+Extend the type above as follows:
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:602]: to me the 'above type' sounds better but up to you...

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 7]
```diff
@@ -663,7 +672,7 @@ a base type can be created with those properties.
 }
 ```
 
-If a new type should be created with properties in addition to the ones shown above,
+If you need to create a new type created with properties in addition to the ones shown above,
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:675]: This doesn't sound right, I would leave it as it was

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 8]
```diff
@@ -728,17 +737,19 @@ The new type may also include the full type definition of the reference type ins
 ```
 
 If the full definition is sent, the referenced types (base type "Simple" in the above example)
-should match the actual type that was initially created. If the full definition is sent and the referenced types do not exist,
-they will be created automatically by the SDS. Further type creations can reference them as shown above.
+should match the actual type that was initially created. 
+If the full definition is sent and the referenced types do not exist,
+new types by the same definition will be created automatically by the SDS.
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:742]: I'd remove this

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md [hunk 9]
```diff
@@ -728,17 +737,19 @@ The new type may also include the full type definition of the reference type ins
...
-Base types and properties of type Object, Enum, or user-defined collections such as Array, List and Dictionary,
+Base types and properties of type `object`, `enum`, or user-defined collections such as `array`, `list` and `dictionary`,
 will be treated as referenced types. Note that streams cannot be created using these referenced types.
-If a stream of particular type is to be created, the type should contain at least one property
+If a stream of a particular type is to be created, the type should contain at least one property
 with a valid index type as described in the [Indexes](xref:sdsIndexes) section. 
 The index property may also be in the base type as shown in the example above.
 
-This works seamlessly when using any programming language, using .NET for example:
+You can do this with any programming languages. Here's an example in .NET:
```
Comment (FreeJules) [inline Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:752]: I'd change this to 'You can do this using any programming language.'

**Review by FreeJules — COMMENTED** [review]
Let's chat about some of these confusing sentences

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md (hunk 2 above)
Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:164]: Type properties are expected to appear in every stream that is created from a given type.

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md (hunk 4 above)
Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:308]: Working with sounds more natural.

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md (hunk 7 above)
Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:675]: If you need to create a new type with additional properties to the ones shown above

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md (hunk 8 above)
Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:742]: replace 742, new type will be created automatically by the SDS

File: Content_Portal/Documentation/SequentialDataStore/SDS_Types.md (hunk 3 above)
Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:193]: Awaiting feedback from Jules and McKenzie

Comment (FreeJules) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:193]: The ``Value`` field is used for properties of enum type. Enums have 'named constants' and those named constants have the associated 'constant values'. When representing an enum as an SdsType, 'Properties' collection is a list of those enums constants. The SdsTypeProperty's ``Id`` represents the constant's name and the SdsTypeProperty's ``Value`` represents the constant's value (see the enum ``State`` definitions in the code samples below).

@mckenzielee does this make more sense?

Comment (FreeJules) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:193]: Final... Replace the whole paragraph under value with flowing: 
The ``Value`` field is used for the properties of enumeration types. An Enum is a collection of named constants with associated constant values. Each named constant is defined by the SdsTypeProperty's Id. The associated constant value is defined by the SdsTypeProperty's Value (see the enum ``State`` definitions in the code samples below).

Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:193]: Is it okay for me to spell out all instances of enum as enumeration _and_ use lowercase?

Comment (FreeJules) [reply Content_Portal/Documentation/SequentialDataStore/SDS_Types.md:193]: Either one is ok.

**Pull request merged**